mod-calendar/src/main/resources/bama/mod-calendar/features/calendarPeriods.feature

```diff
@@ -41,11 +41,14 @@ Feature: Calendar periods
     And param startDate = '2120-08-30'
     When method GET
     Then status 200
-    And match $.totalRecords == 1
```

Comment:
Updated API will return dates inclusive for a calendar's entire range, rather than only those between open days (when a `startDate`/`endDate` is not specified).  In practice, every request made includes these parameters, so this is not breaking.

For example, if a calendar spanned a single Sunday-Saturday week, however, was only open on Monday and Friday, this endpoint would return objects for every day between Monday and Friday, however, skip closed edges on the boundary of the calendar.  This is now updated to be consistent with the boundary of the calendar itself.

Initially, the example calendar went from August 1 to August 31, however, the last open day was August 30, so this endpoint treated the calendar as ending on the 30th (despite this not being true).